main.go

```diff
@@ -84,6 +84,7 @@ func main() {
 				configuration.GetPostgresSSLMode(),
 			))
 		if err != nil {
+			db.Close()
```

Comment:
A call to Close on a failed Open needs to be guarded:

```go
db, err := gorm.Open("postgres")
if err != nil {
	db.Close()
}
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x30 pc=0x601468]

goroutine 10 [running]:
panic(0xbd5460, 0xc820010190)
	/home/aslak/.gvm/gos/go1.6.2/src/runtime/panic.go:481 +0x3e6
testing.tRunner.func1(0xc820022900)
	/home/aslak/.gvm/gos/go1.6.2/src/testing/testing.go:467 +0x192
panic(0xbd5460, 0xc820010190)
	/home/aslak/.gvm/gos/go1.6.2/src/runtime/panic.go:443 +0x4e9
github.com/almighty/almighty-core/vendor/github.com/jinzhu/gorm.(*DB).Close(0xc8202e6d20, 0x0, 0x0)
```

```